mbedtls/src/pk/mod.rs

```diff
@@ -40,9 +38,6 @@ pub use crate::ecp::EcGroup;
 
 pub use dhparam::Dhm;
 
-// SHA-256("Fortanix")[:4]
-const CUSTOM_PK_TYPE: pk_type_t = 0x8b205408 as pk_type_t;
```

Comment:
Why was this removed? We are using custom types.

Reply:
Because custom pk type is no longer available in mbedtls 3.X
So we need to implement it on out end.